users.js

```diff
@@ -1231,41 +1340,44 @@ User = (function () {
 			this.updateIdentity(roomid);
 		}
 	};
-	User.prototype.ban = function (noRecurse, userid) {
+	User.prototype.ban = function (noRecurse, userid, expiration) {
```

Comment:
It should also go before `userid`, since `userid` is mostly internal and `expireTime` is external.